js/ai/src/prompt.ts

```diff
@@ -395,6 +395,7 @@ function promptMetadata(options: PromptConfig<any, any, any>) {
   return {
     ...options.metadata,
     prompt: {
+      ...options.metadata?.prompt,
```

Comment:
I don't know the history here, but copying existing metadata.prompt as the base, and then overriding with the few options below that were already present.